kloppy/infra/serializers/tracking/metrica_epts/metadata.py

```diff
@@ -227,21 +227,29 @@ def load_metadata(
     score_path = objectify.ObjectPath(
         "Metadata.Sessions.Session[0].MatchParameters.Score"
     )
-    score_elm = score_path.find(metadata)
-    score = Score(
-        home=score_elm.LocalTeamScore, away=score_elm.VisitingTeamScore
-    )
-
-    _team_map = {
-        score_elm.attrib["idLocalTeam"]: Ground.HOME,
-        score_elm.attrib["idVisitingTeam"]: Ground.AWAY,
-    }
 
     _team_name_map = {
         team_elm.attrib["id"]: str(team_elm.find("Name"))
         for team_elm in metadata.find("Teams").iterchildren(tag="Team")
     }
 
+    if score_path.hasattr(metadata):
+        score_elm = score_path.find(metadata)
+        score = Score(
+            home=score_elm.LocalTeamScore, away=score_elm.VisitingTeamScore
+        )
+
+        _team_map = {
+            score_elm.attrib["idLocalTeam"]: Ground.HOME,
+            score_elm.attrib["idVisitingTeam"]: Ground.AWAY,
+        }
+    else:
+        score = Score(home=0, away=0)
+        _team_map = {
```

Comment:
Could you replace this with something like
`home_team_id, away_team_id = list(_team_name_map.keys())`

Reply:
And does it makes sense to always use this logic instead of the 
```python
        _team_map = {
            score_elm.attrib["idLocalTeam"]: Ground.HOME,
            score_elm.attrib["idVisitingTeam"]: Ground.AWAY,
        }
```
code?

Reply:
Okee that makes sense. Thanks.

Could you change the `home_team_id, away_team_id = list(_team_name_map.keys())` part? Than I'm able to merge the PR.

Reply:
Done! :)